operator/apis/loki/v1/lokistack_types.go

```diff
@@ -1097,6 +1101,8 @@ const (
 	ReasonZoneAwareNodesMissing LokiStackConditionReason = "ReasonZoneAwareNodesMissing"
 	// ReasonZoneAwareEmptyLabel when the node-label used for zone-awareness has an empty value.
 	ReasonZoneAwareEmptyLabel LokiStackConditionReason = "ReasonZoneAwareEmptyLabel"
+	// ReasonStorageSchemaVersionIsOld when the object storage schema version is older than V13
+	ReasonStorageSchemaVersionIsOld LokiStackConditionReason = "ReasonStorageSchemaVersionIsOld"
```

Comment:
At some point we started using the `Reason` prefix in the value as well, I don't think we need to do that, though:

```suggestion
	ReasonStorageNeedsSchemaUpdate LokiStackConditionReason = "StorageNeedsSchemaUpdate"
```